tox.ini

```diff
@@ -40,3 +48,9 @@ extras=
     test
 commands =
     pylint -E src/ssort tests setup.py
+
+[testenv:rst_linter]
+deps =
+    git+https://github.com/Lucas-C/pre-commit-hooks-markup.git
+commands =
+    rst_linter README.rst
```

Comment:
It looks like `rst_linter` just calls straight out to `readme_renderer` from https://pypi.org/project/readme-renderer/ and checks for errors.  `readme-renderer` is published by the pypa, and so is at least as trustworthy as any other package on pypi.

I would say either pin a particular commit, or replace the command with:
```bash
python -m readme_renderer README.rst -o /dev/null
```

Reply:
i will pin deps to commit

readme_renderer don't have example on how to test if rst file is correct